src/evm_transition_tool/transition_tool.py

```diff
@@ -293,32 +258,32 @@ def collect_traces(
                 traces.append(tx_traces)
         self.append_traces(traces)
 
+    @dataclass
+    class ToolData:
```

Comment:
Looks good to me. I think I prefer the name `TransitionToolData` to make it clear it's only used in `t8n`. Also with a view to a future clean-up regarding the misplaced `verify_fixture()` function in a "`TransitionTool`" class.